docs/virtual_machines/net_binding_plugins/.pages

```diff
@@ -0,0 +1,2 @@
+nav:
+  - network_binding_plugins.md
```

Comment:
Add the two plugins to the list so they appear in the menu

Reply:
They are accessible from the main one. Exposing the list from the main menu is a bit "heavy" IMO.
But if you think it looks better, I do not mind.

Reply:
I think it would be better for visibility. But if you'd rather keep them hidden from the menu, then we should change the configuration so "Network Binding Plugins" does not appear as its own section. If you decide for the latter, then I think you would need to go back to the previous version without .pages in the subdirectory and use `net_binding_plugins/passt.md` as the link (can't find a doc reference to confirm that that works ATM)

Reply:
> then I think you would need to go back to the previous version without .pages in the subdirectory and use `net_binding_plugins/passt.md` as the link (can't find a doc reference to confirm that that works ATM)

That is what I had and it did not work.

Reply:
The one I saw you had it set to 
![image](https://github.com/kubevirt/user-guide/assets/3493830/92fe734d-fd55-481b-b600-9871198c43fc)

have you tried setting it without the leading `./`?

Reply:
I tried that, it did not work.
I ended up with the main file outside and all the specific plugins in the folder.
Hope this is ok with you.

Done.